lib/cylc/task_outputs.py

```diff
@@ -138,6 +138,11 @@ def set_all_completed(self):
         for value in self._by_message.values():
             value[_IS_COMPLETED] = True
 
+    def set_incomplete(self, message):
+        """Set output message to incomplete."""
+        if message in self._by_message:
+            self._by_message[message][_IS_COMPLETED] = False
```

Comment:
Should just use `self.set_completed(message, is_completed=False)`?

Reply:
OK I'll refactor these completion methods slightly...